recipes-init/openrc/openrc_0.54.2.bb

```diff
@@ -12,12 +12,13 @@ SRC_URI = " \
 
 S = "${WORKDIR}/git"
 
-inherit meson
+inherit pkgconfig meson
```

Comment:
Pretty sure this isn't necessary.  It's for making sure that the right `pkg-config`, PKG_CONFIG_PATH, etc are used, not for `PACKAGECONFIG` which is a recipe specific thing.

Reply:
Meson fails locating the library without pkg-config as far as I seen.

Reply:
See[1]:

```
cap_dep = dependency('libcap', version: '>=2.33', required : get_option('capabilities'))
```

```
The dependency detector works with all libraries that provide a pkg-config file. Unfortunately several packages don't provide pkg-config files. Meson has autodetection support for some of these, and they are described [later in this page](https://mesonbuild.com/Dependencies.html#dependencies-with-custom-lookup-functionality).
```

The `libcap` is the only one with version constraint, so you enjoyed the fallback to "auto detection" so far.

[1] https://mesonbuild.com/Dependencies.html

Reply:
Right you are, thanks for the explanation.